x-pack/filebeat/input/s3/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package s3
+
+import (
+	"time"
+
+	"github.com/elastic/beats/filebeat/harvester"
+	awscommon "github.com/elastic/beats/x-pack/libbeat/common/aws"
+)
+
+var defaultConfig = config{
+	ForwarderConfig: harvester.ForwarderConfig{
+		Type: "s3",
+	},
+	VisibilityTimeout: 300 * time.Second,
+}
```

Comment:
nit;
1. put types before var and functions
2. transform `defaultConfig` into a function: `func defaultConfig() config { ... }`. This reduces the chance of errors and unwanted sharing if someone ever introduces a type that will hold data by reference.